### DISCUSSION TITLE
Open Badges?

### DISCUSSION THREAD

**Discussion by benel** [community/community#19810] [Other Feature Feedback, Questions, & Ideas]
- Wouldn't it be cool to collect and share GitHub badges along with badges from other issuers?
- Hence, is there a plan to comply with [OpenBadges](https://en.wikipedia.org/wiki/Mozilla_Open_Badges)?

**Comment by JhonnyLusonode** [comment]
Hello @benel 
That might be a great idea! 👍 +1

**Comment by noActive** [comment]
Hi @benbel
I'm for!
great idea

**Comment by Kayke-Fujinaka** [comment]
Great idea
